src/backend/commonTypes.ts

```diff
@@ -17,6 +17,7 @@ export interface Config {
     accountsToScrape: AccountToScrapeConfig[];
     chromiumPath?: string;
     maxConcurrency?: number;
+    timeout:number;
```

Comment:
```suggestion
    timeout: number;
```

Reply:
Thanks,
Strange that the eslint did not take care of it for me as it does on the client side.
Fixed.